worker/fakes/fake_container.go

```diff
@@ -473,6 +481,38 @@ func (fake *FakeContainer) CurrentCPULimitsReturns(result1 garden.CPULimits, res
 	}{result1, result2}
 }
 
+func (fake *FakeContainer) LimitDisk(limits garden.DiskLimits) error {
```

Comment:
These shouldn't be re-added; make sure your local checkout of `cloudfoundry-incubator/garden` is the latest.